src/common/models/Projection.ts

```diff
@@ -403,10 +413,13 @@ export class Projection {
       currDate = currDate.clone().add(1, 'days');
     }
 
+    // only keep futureDaysToInclude days ahead of today
+    const now = new Date();
+    const days = dates.findIndex(date => date > now) + futureDaysToInclude;
```

Comment:
I was going to point out that we should do `_.findIndex(dates, date => date > now)` if we care about IE support, but it looks like the site is already broken on IE, so I guess we don't care. 😁